resilience4j-framework-common/src/main/java/io/github/resilience4j/common/retry/configuration/RetryConfigurationProperties.java

```diff
@@ -231,49 +252,62 @@ private void configureRetryIntervalFunction(InstanceProperties properties,
 
         @Nullable
         private Integer maxAttempts;
-        /*
+
+        /**
          * retry exception predicate class to be used to evaluate the exception to retry or not
          */
         @Nullable
         private Class<? extends Predicate<Throwable>> retryExceptionPredicate;
-        /*
+
+        /**
          * retry setResultPredicate predicate class to be used to evaluate the result to retry or not
          */
         @Nullable
         private Class<? extends Predicate<Object>> resultPredicate;
-        /*
+
+        /**
          * list of retry exception classes
          */
         @SuppressWarnings("unchecked")
         @Nullable
         private Class<? extends Throwable>[] retryExceptions;
-        /*
+
+        /**
          * list of retry ignored exception classes
          */
         @SuppressWarnings("unchecked")
         @Nullable
         private Class<? extends Throwable>[] ignoreExceptions;
-        /*
+
+        /**
          * event buffer size for generated retry events
          */
         @Nullable
         private Integer eventConsumerBufferSize;
-        /*
+
+        /**
          * flag to enable Exponential backoff policy or not for retry policy delay
          */
         @Nullable
         private Boolean enableExponentialBackoff;
-        /*
+
+        /**
          * exponential backoff multiplier value
          */
         private Double exponentialBackoffMultiplier;
 
-        /*
+        /**
+         * exponential max interval value
+         */
+        private Long exponentialMaxIntervalMillis;
```

Comment:
Can we use Duration and call it `maxWaitDuration`?

Reply:
good idea, done